docs/pages/inboxes/history-sync.mdx

```diff
@@ -54,6 +54,8 @@ History syncs are accomplished using these components:
 
 A sync group is a special [Messaging Layer Security](/protocol/security) group that includes all of the user’s devices. The sync group is used to send serialized sync messages across the user's devices. The sync group is filtered out of most queries by default so they don’t appear in the user’s inbox.
 
+You can sync a sync group independent of the history sync process. To learn more, see [Sync a sync group](/inboxes/sync-and-syncall#sync-a-sync-group).
```

Comment:
I wonder if we should call it something other than a sync group. Like "you can sync state between multiple installations independent of history sync process."